_squirtle/mod.ts

```diff
@@ -406,11 +437,23 @@ const runSubmit = async (argv: any) => {
 }
 
 const runRPC = async (argv: any) => {
```

Comment:
I think I'd like all this rpc stuff split out into it's own PR. It's fine if it's dependent on this PR getting merged but this is an entirely separate interest than the rest of this PR and should be able to be reviewed independent of the other tooling and plumbing tweaks